### PR TITLE
mini doesn't require DB

### PR DESCRIPTION
**Pull request by KyleFin**

### CODE REVIEW

File: versioned_docs/version-2.0.0/06-deployments/05-general.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ You can host Serverpod anywhere, running Dart directly or through a Docker conta
 
 ## Required services
 
-Serverpod will not run without a link to a Postgres database with the correct tables added. Serverpod can also optionally use Redis. You enable Redis in your configuration files.
+Serverpod will not run without a link to a Postgres database with the correct tables added (unless you're running Serverpod mini). Serverpod can also optionally use Redis. You enable Redis in your configuration files.
```
Comment (KyleFin) [inline versioned_docs/version-2.0.0/06-deployments/05-general.md:7]: Is this understanding correct? (Based on https://medium.com/serverpod/getting-started-with-serverpod-mini-58c71ffefa54)

Comment (SandPod) [reply versioned_docs/version-2.0.0/06-deployments/05-general.md:7]: Yes, this is correct! 👍

**Review by SandPod — CHANGES_REQUESTED** [review]
Another good catch, please also apply it to the non versioned docs, located here `docs/06-deployments/05-general.md`, so that the change will be preserved once we release a new version of our documentation.

**Review by SandPod — APPROVED** [review]
🚀

**Pull request merged**